### DISCUSSION TITLE
Derivative column like Rowy.io

### DISCUSSION THREAD

**Discussion by ebauger** [supabase/supabase#8673] [Feature Requests]
Hello supabase team!

I tested recently [rowy.io](https://github.com/rowyio/rowy) based on Google Cloud infrastructure. You can watch the excellent video episode [Learn With Jason](https://youtu.be/-WHS2vyjq88?t=2352) explain how using it from the developer perspective.

It will be very interesting to merge the powerful Database function and Edge Function for creating a more turn-key solution to create derivative columns using Deno engine (and with node.js with the require/import auto install packages like rowy).

 Some key feature:
- The user can create derivative script (in js/ts) (like Rowy) on the UI table of Supabase;
- Supabase can create and commit script in a git repository for tracking changes for a developer experience.
- Possibility to observe columns instead of operation like INSERT or UPDATE.

I don't know the codebase of your project and what it’s the scope for implement it, but it will a killer feature for supabase!

Here a [demo](https://demo.rowy.io/tables) where you can try rowy without a subscription. 

Sound promising or it's not something you want to include on your roadmap?

**Comment by EmmanuelTheCoder** [comment]
This sounds great. I wonder why no one is talking about it yet.
